tests/test_visitors/test_ast/test_naming/test_class_attributes.py

```diff
@@ -52,7 +53,7 @@ def test_too_short_attribute_names(
     assert_errors, parse_ast_tree, short_name, code, default_options,
 ):
     """Testing that attribute can not have too short names."""
-    tree = parse_ast_tree(code.format(short_name))
+    tree = parse_ast_tree(code.format(short_name.lower()))
```

Comment:
It is better to use https://docs.python.org/2/library/string.html#string.ascii_lowercase here

Reply:
Okay.